src/forms/userForm.tsx

```diff
@@ -0,0 +1,372 @@
+                    updatedPermissions = {
+                      model_management: this.state.permissions.model_management,
+                      asset_management: this.state.permissions.asset_management,
+                      power_control: this.state.permissions.power_control,
+                      audit_read: this.state.permissions.audit_read,
+                      admin: !this.state.permissions.admin,
+                      datacenter_permissions: this.state.permissions
+                        .datacenter_permissions
+                    };
+                    this.setState({
+                      permissions: updatedPermissions
+                    });
+                  }}
+                ></Checkbox>
+              </FormGroup>
+              <FormGroup inline={true}>
+                <Checkbox
+                  label="Model Management"
+                  alignIndicator={Alignment.LEFT}
+                  checked={
+                    this.state.permissions.admin
+                      ? true
+                      : this.state.permissions.model_management
+                  }
+                  disabled={this.state.permissions.admin}
+                  onChange={() => {
+                    this.setState({
+                      permissions: this.updateBooleanPermissions(
+                        "model_management"
+                      )
+                    });
+                  }}
+                ></Checkbox>
+              </FormGroup>
+              <FormGroup inline={true}>
+                <Checkbox
+                  label="Power Control"
+                  alignIndicator={Alignment.LEFT}
+                  checked={
+                    this.state.permissions.admin
+                      ? true
+                      : this.state.permissions.power_control
+                  }
+                  disabled={this.state.permissions.admin}
+                  onChange={() => {
+                    this.setState({
+                      permissions: this.updateBooleanPermissions(
+                        "power_control"
+                      )
+                    });
+                  }}
+                ></Checkbox>
+              </FormGroup>
+              <FormGroup inline={true}>
+                <Checkbox
+                  label="Audit Log"
+                  checked={
+                    this.state.permissions.admin
+                      ? true
+                      : this.state.permissions.audit_read
+                  }
+                  alignIndicator={Alignment.LEFT}
+                  disabled={this.state.permissions.admin}
+                  onChange={() => {
+                    this.setState({
+                      permissions: this.updateBooleanPermissions("audit_read")
+                    });
+                  }}
+                ></Checkbox>
+              </FormGroup>
+              <RadioGroup
+                inline={true}
+                label="Datacenter permissions"
```

Comment:
can you label this "Asset Management" instead? that's how it's defined in the requirements

Reply:
Done!